components/ui/header.tsx

```diff
@@ -5,41 +5,99 @@ import PearLightLogo from "./PearLight70x70.svg";
 
 export default function Header() {
   return (
-    <header className="absolute z-30 w-full">
-      <div className="mx-auto max-w-6xl px-4 sm:px-6">
-        <div className="flex h-20 items-center justify-between">
+    <header className="animate-fadein-opacity fixed top-0 z-30 w-full bg-white-50 md:bg-transparent md:backdrop-blur-sm">
+      <div className="mx-auto max-w-screen-xl px-4 py-4 sm:px-6">
+        <div className="roboto-medium flex h-20 items-center justify-between text-2xl text-black transition ease-in-out hover:text-gray-600 md:text-xl">
```

Comment:
- can remove `roboto-medium` since font is still not decided
- `hover:text-gray-600` is applied to the parent `<div>` which wraps the entire navbar content this means that when you hover anywhere within that `<div>` including the middle of the navbar the hover styles will be triggered and applied to all the child elements, including the text. To fix this you can move the hover:text-gray-600 class to the specific elements that you want to be affected by the hover state such as the individual navigation links

Reply:
thats totally fine i din wanted to implement it at first place

Reply:
yea i realize it too now, done